journal/modernized/35-chapter-32.adoc

```diff
@@ -779,7 +780,8 @@ and to the patient waiting for Christ, that they might know Him to be in them,
 "`who lives and abides forever.`"
 The above is the substance of what I had to express among them;
 and though some individuals mocked and behaved rudely, saying aloud,
-'`I wish he had doneso that James Watkin stopped, and reproved them for their behaviour;
+"`I wish he had done;`" so that James Watkin stopped, and reproved them for their behaviour;
+//I looked this up in the PDF and changed it to how it was there because I'm thinking it's the broken English again.
```

Comment:
yeah, this is an archaic idiom that you'll see a lot in these books.  "had done" they used for "was finished" or "had finished".  George Fox says it all the time in his journal, like "I went into the steeple-house, and when the priest had done, I spoke to the people" -- meaning "when the priest had finished".

So, I think you did the right thing correcting the quotation marks to match the original. But let's also change the expression here, but only in the modernized.  Let's make it:

```ASCIIDOC
"`I wish he would finish;`" so that James Watkin stopped, and reproved them for their behaviour;
```